website/src/scss/index.scss

```diff
@@ -0,0 +1,21 @@
+/* stylelint-disable docusaurus/copyright-header */
+/**
+ * CSS files with the .module.css suffix will be treated as CSS modules
+ * and scoped locally.
+ */
+
+.footer {
+  box-shadow: none;
+  margin-top: 30px;
+
+  &__heading {
+    text-align: center;
+  }
+
+  &__title {
+    font-size: 64px;
+    font-weight: bold;
+    line-height: 1.03;
+    letter-spacing: -2px;
```

Comment:
Could you somehow extract this?

Ideally (if there's any semblance of consistency in the design), you'd put all typography styles in one shared map. Then you'd also add a `$footer` map on top of this file and make it use the values from the global map.

Or at least extract it into a `$footer` map at the too of the file.

Reply:
Seeing as you already have `footer.scss`, you can move this code to that file.